spec/factories/courses.rb

```diff
@@ -118,6 +118,9 @@
     meta { nil }
     age_range_in_years { "11_to_16" }
     program_type { "pg_teaching_apprenticeship" }
+    degree_grade { "two_one" }
+    additional_degree_subject_requirements { true }
```

Comment:
```ruby
options.sample
```
For the next person that stumbles here and wonders what the options are rather then referring back to spec/components/degree_row_content_component_spec.rb

Reply:
THe problem with this is if `additional_degree_subject_requirements ` is true then 'degree_subject_requirements' should have a value. Seems like a bit of hassle to add an after create action just for context when perusing the factory. Will do for grade though!